docs/pages/connect-your-client/vnet.mdx

```diff
@@ -13,12 +14,13 @@ Teleport. Underneath, VNet authenticates the connection with your credentials, u
 routing technology](../architecture/tls-routing.mdx) as `tsh proxy app`. This is all done
 client-side – VNet sets up a local DNS name server and a virtual network device.
 
-VNet is available in Teleport Connect and tsh on macOS, with Windows and Linux version in the works.
+VNet is available in Teleport Connect and `tsh` on macOS, with Windows and Linux version in the works.
 
 ## Prerequisites
 
 - A client machine running macOS Ventura (13.0) or higher.
 - [Teleport Connect](teleport-connect.mdx), version 16.0.0 or higher.
+- TCP Application resources available to your user.
```

Comment:
AFAIK the guides under Connect your Client are meant for end users, so I think it should say something like "A TCP application available in the cluster".